controller/api/public/stat_summary_test.go

```diff
@@ -958,12 +958,11 @@ status:
 													},
 													TimeWindow: "1m",
 													Stats: &pb.BasicStats{
-														SuccessCount:    123,
-														FailureCount:    0,
-														LatencyMsP50:    123,
-														LatencyMsP95:    123,
-														LatencyMsP99:    123,
-														TlsRequestCount: 123,
+														SuccessCount: 123,
```

Comment:
Mind retaining the original formatting for your changes in `stat_summary_test.go` and `test_helper.go`? That makes it easier to see what has changed, and prevents introducing new errors.

Reply:
Oops, I see why you changed the formatting now -- removing `TlsRequestCount` made it not necessary to have those extra spaces. No need to change it back, good call 👍